src/index.ts

```diff
@@ -31,7 +31,8 @@ const parseOptions = {
  * https://github.com/conventional-changelog/conventional-changelog/tree/master/packages/conventional-changelog-angular#type
  */
 const releaseRulesExtension = [
-  { release: 'patch', scope: 'deps', type: 'chore' },
+  { release: 'patch', type: 'chore' },
+  { release: 'patch', type: 'ci' },
```

Comment:
Can you at least for the time being add the following rules temporarily until we implement rule override:
```
{ release: false, scope: 'deps-dev', type: 'chore' }
{ release: 'patch', scope: 'deps-dev', type: 'chore', message: '*typescript*' }
```

This will prevent all the packages which use release-me from releasing on dev depedencies which add 0 value to the actual release package.

Reply:
That's not true. You have many dev dependencies that result in changes to build output: Protocol Buffer generators, Flatbuffers generators, not even going to list hundreds of front end dependencies that do that.

Reply:
Then we are reverting back to having releases for every dev-depedency triggering functions to redploying incurring more and more costs for no reason. For example https://github.com/ridedott/id/releases

This project is unchanged in such a long time, but if it would release on each package update, it would then trigger dependency update in functions once again, triggering release once again.

I can agree with adding chore without a scope, but I cannot approve allowing all chores for dev dependency to trigger releases once again causing cascaded releases.

Reply:
I'm gonna merge it for now, until we have time to implement custom filters. I'm against adding any filters by default, it's not beneficial for the Community. We can't know who's going to use this package and for what purpose, having these assumptions is dangerous.